code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -147,7 +147,7 @@
 	if(istype(target, /obj/machinery/disposal))
 		return FALSE
 
-	var/resolved = target.attackby(device)
+	var/resolved = target.attackby(device, holder.wearer)
```

Comment:
I think this will produce the wrong logs, if the wearer is in a suit with an AI/pAI too, and the AI/pAI attacks something.